docker/supabase/volumes/db/init/04-telescope-schema.sql

```diff
@@ -0,0 +1,41 @@
+-- Case-insensitive text type
+CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA extensions;
+
+-- Registered Telescope users
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  display_name citext UNIQUE NOT NULL,
+  first_name text NOT NULL,
+  last_name text NOT NULL,
+  email citext UNIQUE NOT NULL,
+  github_username citext UNIQUE NOT NULL,
+  github_avatar_url text NOT NULL
+);
+
+-- Planet CDOT Feed List, see https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List
+CREATE TABLE IF NOT EXISTS feeds (
+  url text PRIMARY KEY,
+  user_id text references telescope_profiles ON DELETE CASCADE, -- optional, a user can claim an existing feed when they register
```

Comment:
The `ON DELETE CASCADE` means that when a user is deleted, their feed would also be deleted, or is it the other way around?

Reply:
`ON DELETE CASCADE` would mean if we delete a telescope profile, the feed connected to that profile will also be deleted. So a hypothetical scenario, I sign up as a Telescope user but I'm a baddie and I claim a feed that's obviously not mine, such as:

```
[https://epicgamercharlie.wordpress.com/feed/]
name = Charlie Le
```

then I delete my telescope profile and Charlie's feed would get deleted as well. 

I pick on Charlie because we had a couple of classes together and I was `cle15` and he was `cle17`. My arch-nemesis.

Reply:
This means feeds will be deleted if a user is deleted. The default behavior would prevent user deletion if a feed is referencing its primary key.

Reply:
@cindyledev, anyone can go and claim feeds + GitHub account that's not theirs, it's built on trust pretty much. Maybe we should file an issue to prevent 'baddies' in the future.

Reply:
A great lazy way would be to only allow admins to delete feeds because who will stop me from committing such a sin? >:D

Reply:
Every user is already a Seneca user, and if someone does something bad, we'll deal with them directly.

I don't think we have to worry about anonymous trolls doing weird things here.

Reply:
When this PR is in, there's will be another PR for RLS of these two tables which restrict certain queries can be done on by users.